decryptDefaultKey/decryptDefaultKey.ino

```diff
@@ -0,0 +1,82 @@
+#include <AESLib.h>
+
+#define BAUD 9600
+
+AESLib aesLib;
+String kim="r9Huf1ov92qFJMtG";
```

Comment:
how you arrive this key?

Reply:
kim?

Reply:
ok will use meaning variable names and will not use default key